src/compose/network.ts

```diff
@@ -193,7 +193,7 @@ export class Network {
 				.getNetwork(Network.generateDockerName(this.appId, this.name))
 				.remove(),
 		).tapCatch(error => {
-			this.logger.logSystemEvent(logTypes.createNetworkError, {
+			this.logger.logSystemEvent(logTypes.removeNetworkError, {
 				network: { name: this.name, appId: this.appId },
```

Comment:
is this right?

Reply:
How do you mean?

Reply:
Ok - reason for the false flag; the PR title is "... cannot remove a volume" and this is all about networks...

Reply:
Ah yeah, accidentally put volume instead of network in the commit message - changed now. Nice catch!